go/cmd/gh-ost/main.go

```diff
@@ -127,6 +128,10 @@ func main() {
 
 	flag.UintVar(&migrationContext.ReplicaServerId, "replica-server-id", 99999, "server id used by gh-ost process. Default: 99999")
 
+	flag.BoolVar(&migrationContext.IncludeTriggers, "include-triggers", false, "When true, the triggers (if exist) will be created on the new table")
+	flag.StringVar(&migrationContext.TriggerSuffix, "trigger-suffix", "", "have to be used with '--include-triggers'")
+	flag.BoolVar(&migrationContext.RemoveTriggerSuffix, "remove-trigger-suffix-if-exists", false, "Removal (instead of addition) of trigger suffix if it exists at the end of the original triger name. Have to be used with '--include-triggers' and '--TriggerSuffix'")
```

Comment:
`TriggerSuffix` is a typo, should be `trigger-suffix`

```suggestion
	flag.BoolVar(&migrationContext.RemoveTriggerSuffix, "remove-trigger-suffix-if-exists", false, "Remove given suffix from name of trigger. Requires '--include-triggers' and '--trigger-suffix'")
```